flocker/node/script.py

```diff
@@ -438,7 +439,7 @@ class BackendDescription(PClass):
     # out.
     needs_cluster_id = field(type=bool, mandatory=True)
     # Config "dataset" keys required to initialize this backend.
-    required_config = field(type=set, mandatory=True, initial=set())
+    required_config = pset_field(unicode)
```

Comment:
Sorry, thinking about this again... it'd really be annoying if someone develops a driver with flocker 1.10 and forgets to put the "u" in front of the strings, and then later upgrades and their plugin breaks.

Could we add a factory to this field that will turn "str" values into "unicode" or instead just not specify a type of the set?